src/pages/cli/custom/cdk.mdx

```diff
@@ -18,31 +18,48 @@ amplify add custom
 
 A skeleton CDK stack is generated in the `amplify/backend/custom/<resource-name>` directory. Edit the `cdk-stack.ts` file to define the additional AWS resources. Add additional CDK constructs using the `package.json`. Run `amplify build` to download all NPM dependencies for this custom resource and synthesize the CDK stack.
 
+<Callout>
+
+To migrate from AWS CDK version v1 to v2, refer to the [migration documentation](/cli/migration/aws-cdk-migration/).
```

Comment:
```suggestion
To migrate from AWS CDK v1 to v2, refer to the [migration documentation](/cli/migration/aws-cdk-migration/).
```

remove redundant "version" since we are referring to versions as "v1", "v2"